include/capstone/mips.h

```diff
@@ -244,6 +687,12 @@ typedef struct cs_mips_op {
 		int64_t imm;		///< immediate value for IMM operand
 		mips_op_mem mem;	///< base/index/scale/disp value for MEM operand
 	};
+	bool is_reglist;    ///< defines if the register is part of a list
+	bool is_unsigned;   ///< when true, the immediate value is unsigned
```

Comment:
Are there `uint64_t` possible in Mips? If yes, the immediate value above should be of the type. Otherwise this is fine.

Reply:
there is no difference for capstone since type IMM is always int64, so what can change?